bin/xud

```diff
@@ -87,6 +87,16 @@ const { argv } = require('yargs')
       type: 'boolean',
       default: undefined,
     },
+    'p2p.discover': {
+      describe: 'Do not send GET_NODES if false, when connecting to a peer',
```

Comment:
I don't think we need the internal packet terminology here, we can just say "Automatically discover and connect to new nodes"